src/allocator/mod.rs

```diff
@@ -0,0 +1,93 @@
+// use crate::arch;
+extern crate spin;
```

Comment:
We don't need to mark extern crates as extern anymore, but I'm not against it if it is for clarity

Reply:
We can just
 ``` rust
use spin::....
```
?

Reply:
Yep, extern is not needed anymore since Rust 2018

Reply:
Oups